src/System.IO.FileSystem/src/System/IO/UnixFileSystemObject.cs

```diff
@@ -178,7 +178,14 @@ public DateTimeOffset CreationTime
                         DateTimeOffset.FromUnixTimeSeconds(_fileinfo.btime) :
                         default(DateTimeOffset);
                 }
-                set { } // Not supported
+                set 
+                {
+                    // The ctime in Unix can be interpreted differently by different formats so there isn't
+                    // a reliable way to set this; however, we can't just do nothing since the FileSystemWatcher
+                    // specifically looks for this call to make a Metatdata Change, so we should rev the 
+                    // LastAccessTime of the file to cause the metadata change we need.
+                    LastAccessTime = DateTimeOffset.Now;
```

Comment:
Is a call to utime sufficient to trigger the event, or do we actually need to change the value?  I'm not loving actually changing the last access time if we can avoid it.  In other words, would this be sufficient to trigger the event?
```C#
LastAccessTime = LastAccessTime;
```
?